lib/matplotlib/__init__.py

```diff
@@ -225,7 +225,7 @@ def _is_writable_dir(p):
             t.write(b'1')
         finally:
             t.close()
-    except OSError:
+    except:
```

Comment:
don't forget a "# noqa" flag here. I would also think it would be better to do catch Exception as a bare except catches *everything*.